lib/scheduler.ml

```diff
@@ -144,10 +144,15 @@ module Make (Streamd : StreamDescriptor) = struct
       ~initial_recv_window_size
       descriptor
     =
+    let t_last =
+      match parent with
+      | Parent (Connection root) -> root.t_last
+      | Parent (Stream stream) -> stream.t_last
+    in
     Stream
       { descriptor
-      ; t_last = 0
-      ; t = 0
+      ; t_last
```

Comment:
I'm now doubting that my comment [here](https://github.com/anmonteiro/ocaml-h2/pull/199#discussion_r1093661829) is wrong.

@quernd why are new streams created with the parent's `t_last`?

Reply:
reverting this doesn't seem to have an effect on the failing example from #162.

Reply:
This was my reasoning, maybe I'm misunderstanding something:
- If we set `t = 0` then younger streams will have a lower priority on creation than older streams which means that they get an unfair extra "budget" of data. So they should have `t` set to their parent's `t_last`
- For that reason, `t_last` should be inherited from the parent and not set to `0` in case new streams are added before `t_last` was ever updated (if that's possible)

Reply:
I think I agree for `t`. But `t_last` only matters if this stream has children under them. So that one should start at 0, I think.

Reply:
I guess you're right for `t_last`. Priorities are not global so they should start at 0.